lang/semgrep-grammars/src/semgrep-dockerfile/test/corpus/semgrep.txt

```diff
@@ -65,3 +65,380 @@ RUN ["a", $X, "b"]
       (double_quoted_string)
       (semgrep_metavariable)
       (double_quoted_string))))
+
+=============================================
+Param metavariable
+=============================================
+
+FROM --platform=$PLATFORM debian
+ADD --chown=$USER_GROUP src dst
+
+---
+
+(source_file
+  (from_instruction
+    (param)
```

Comment:
this --platform=$PLATFORM is reduced just to a 'param'?

Reply:
yes. The details are not shown due to hidden rules (something like `_param_key`), and we can't unhide them without breaking the tests inherited from the original tree-sitter-dockerfile.